FOPTS-846 Kubecost request rightsizing policy - unable to apply with default parameter values (#1281)
flexera-public/policy_templates: cost/kubecost/sizing/CHANGELOG.md

# Changelog

## v0.2

> - CPU and RAM parameters now based on percentiles.

Reviewer: @vadgun I think that the following could make it more explicit:

```suggestion
- The CPU quantile and Memory quantile parameters are now percentiles.
```

> - Changed to percentile on CPU and RAM to request recommendations on, now receives a number between 1 and 100, default value set to 75 %

Reviewer 2: Correct grammar so that it does not use commas to end sentences. We also don't need to get too detailed in the CHANGELOG as the specifics will be in the README. Example:

- CPU and RAM parameters now based on percentiles.

Author: Solved, thank you
